### PR TITLE
updated input line

### PR DESCRIPTION
**Pull request by sethmh82**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
 					<div class="col-md-6 col-md-offset-3">
 						<!-- SIGN IN FORM -->
 						<div class="container-fluid page-header">
-							<h1>SIGN IN HERE:</h1>
+							<h1>YOU SHOULD DEFINITELY SIGN IN HERE:</h1>
```
Comment (sethmh82) [inline index.html:43]: Are you sure???

Comment (mehdicode) [inline index.html:43]: solved

**Comment by sethmh82** [comment]
pull request 👍

File: index.html (hunk 1 above)
Comment (sethmh82) [reply index.html:43]: okay

**Pull request merged**